stdlib/LinearAlgebra/src/triangular.jl

```diff
@@ -2445,7 +2445,8 @@ Base.@propagate_inbounds function _sqrt_real_2x2!(R, A)
     # avoid overflow/underflow of μ
     # for real sqrt, |d| ≤ 2 max(|a12|,|a21|)
     μₛ = max(abs(a12), abs(a21))
-    μ = μₛ * sqrt(-(d / μₛ)^2 - 4 * (a21 / μₛ) * (a12 / μₛ)) / 2
+    μₛ⁻¹ = inv(μₛ)
```

Comment:
Technically this only works if the scale factor is not subnormal.

Reply:
Hm okay, so division is safer.